Controller/CRUDController.php

```diff
@@ -296,7 +295,6 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      *
-     * @throws NotFoundHttpException If the object does not exist
```

Comment:
I think this was caused by PHPStorm Symfony 2 plugins cause none of access denied of not found exception are auto-documented. Maybe because considered as not necessary.

Anyway, it's done now. :+1: